app/uk/gov/hmrc/apidefinition/config/AppConfig.scala

```diff
@@ -39,7 +39,7 @@ class AppConfig @Inject() (val runModeConfiguration: Configuration, environment:
 
   lazy val apisToRemove = runModeConfiguration.underlying.as[List[String]]("apisToRemove")
 
-  lazy val apisToRetire = runModeConfiguration.underlying.as[List[String]]("apisToRetire")
+  lazy val apisToRetire: Seq[String] = runModeConfiguration.underlying.as[Option[List[String]]]("apisToRetire").getOrElse(List.empty)
```

Comment:
Maybe make this return a `List[String]` since I think there's been some disquiet expressed about this style of code

Reply:
I originally suggested putting the key but no value in application.conf but Anjum suggested what James has done here which I thought was more elegant. 
The obvious place for the config would be app-config-base AFAICS and I always forget to look there. That's partly why I liked having a view of the default value close at hand.